stores-api/mir-api/src/main/java/com/bloxbean/cardano/yaci/store/api/mir/controller/MIRController.java

```diff
@@ -4,23 +4,28 @@
 import com.bloxbean.cardano.yaci.store.mir.domain.MoveInstataneousReward;
 import com.bloxbean.cardano.yaci.store.mir.domain.MoveInstataneousRewardSummary;
 import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.web.bind.annotation.*;
 
 import java.util.List;
 
+@Slf4j
 @RestController
-@RequestMapping("${apiPrefix}")
 @RequiredArgsConstructor
-@Slf4j
+@Tag(name = "Network Service")
```

Comment:
I am not sure if Network Service is the right tag here. 
MIR withdrawals are like treasury management. For now, I think we can call it MIR service.

The second method in this controller which returns MIRs by transaction can be moved to another controller, for example: MIRTxController and we can tag it to "Transaction Service".
Alternatively, we can just tag that method with "Transaction Service" if possible.

Reply:
I think it's good to have both endpoints under MIR Service as they are responding with MIR Objects